api/controllers/linux/install/installation.go

```diff
@@ -70,6 +75,24 @@ func (c *InstallController) configureInstallation(ctx context.Context, config ty
 		return types.NewConflictError(err)
 	}
 
+	defer func() {
+		if finalErr != nil {
+			failureStatus := types.Status{
+				State:       types.StateFailed,
+				Description: finalErr.Error(),
+				LastUpdated: time.Now(),
+			}
+
+			if err = c.store.InstallationStore().SetStatus(failureStatus); err != nil {
+				c.logger.Errorf("failed to update status: %w", err)
+			}
```

Comment:
This status update feels a bit off, since in all the other places (here included) it's the manager responsibility to typically do this. Which makes me wonder if we shouldn't move this entire method to the manager and only deal with error handling/state transitions here.

Reply:
I agree that validate and set could be combined and moved to the manager as well as setting the status. I'm not sure that the manager should be aware of the runtimeconfig

Reply:
discussed irl that it is ok to move rc into the manager

Reply:
Captured in - https://app.shortcut.com/replicated/story/125757/move-configure-installation-to-the-installation-manager